_benchmark/user-guide/contributing-workloads.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Contributing custom workloads
+nav_order: 11
+parent: User guide
+---
+
+# Contributing custom workloads
+
+After [creating a custom workload]({{site.url}}{{site.baseurl}}/benchmark/creating-custom-workloads/), you can contribute your workload for other OpenSearch Benchmark users to use. If you plan on contributing a workload, use the following guidelines.
+
+## README and license
```

Comment:
If these are supposed to be steps in a process, can you rename to "Create a README file", then "Test the workload", then "Raise a pull request". I'm not sure where the Structure section comes into play, though... Should that be at the beginning of the topic?